contentcuration/contentcuration/frontend/shared/views/LicenseDropdown.vue

```diff
@@ -277,4 +284,13 @@
     }
   }
 
+  .help {
+    position: relative;
+    top: -25px;
```

Comment:
Could you put a comment explaining why this is hardcoded like this? It LGTM but since it is a particular case it can be helpful to leave a note about what this is doing on the page.

Reply:
That's a good point, I'll add that in.